github/search.go

```diff
@@ -15,6 +15,13 @@ import (
 // SearchService provides access to the search related functions
 // in the GitHub API.
 //
+// Each method takes a query string defining the search keywords and any search qualifiers. For example,
+// when searching issues, the query "gopher is:issue language:go" will search for issues containing the word "gopher"
+// in go repositories. The method call
```

Comment:
s/in go repositories/in Go repositories/ since the language is called "Go", not "go".